client/app/collections/calendars.coffee

```diff
@@ -88,9 +88,17 @@ module.exports = class CalendarCollection extends TagCollection
 
     toArray: -> @map stringify
 
-    toAutoCompleteSource: -> 
-        return @map (tag) -> 
-            return _.extend 
+    comparator: (a, b) ->
+        if a.get('name') < b.get('name')
```

Comment:
You should use [localeCompare](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) method to compare two strings.